### DISCUSSION TITLE
indent-blankline errors

### DISCUSSION THREAD

**Discussion by moonstruckmoth** [LazyVim/LazyVim#1513] [General]
It seems indent-blankline.nvim has just moved to [version 3](https://github.com/lukas-reineke/indent-blankline.nvim/commit/13e415c6f70fed2f37162c2fbfd07b33d144e15b) which is throwing errors in LazyVim,

Error  10:57:00 AM notify.error You are trying to call the setup function of indent-blankline version 2, but you have version 3 installed.

TIA

**Comment by folke** [comment]
Lazyvim was already updated

**Reply by moonstruckmoth** [reply]
Thanks!
